shared/utils/relabel-ids.py

```diff
@@ -201,15 +201,36 @@ def _add_cce_id_refs_to_oval_checks(self, idmappingdict):
                 msg = "Failed to add CCE ID to {0}.".format(ovalid)
                 raise ssgcommon.SSGError(msg)
 
+    def get_nested_definitions(self, oval_def_id):
+        processed_def_ids = set()
+        queue = set([oval_def_id])
+        while True:
```

Comment:
You can have `while queue:` here, and get rid of the try/except when popping.

Reply:
Thanks, looks much better.